### ISSUE TITLE
New APK doesnt support encryption

### ISSUE THREAD

**Issue by Bright1G** [OTCv8/otclientv8#201]
Hello,
so a lot of people are having this new issues with latest APK, so for some reason it doesnt support encryption anymore and you getting fatal error, the only way it works if you use otclientv8_gl.exe --encrypt --android but if u decide to use it, your ingame bot will crash the client every single time because it cant decrypt your bot, so the only solution right now is to use old apk files that im doing right now and everything works just fine its only that i dont have the latest features because of it. So it would be great if this issue would be solved asap

**Comment by Oen44** [comment]
Literally nothing related to encryption was touched in a long time.

**Comment by Bright1G** [comment]
Thats actually super weird because older apk works just perfectly but latest one gives fatal error saying unable to load corelib

**Comment by especialized-oldschoool** [comment]
im having the exact same issues lately

**Comment by zafrim** [comment]
This problem has not yet been fixed. still happens

**Comment by Oen44** [comment]
> This problem has not yet been fixed. still happens

It was. Errors related to bot module when encrypting was fixed months ago.

**Comment by zafrim** [comment]
> > This problem has not yet been fixed. still happens
> 
> It was. Errors related to bot module when encrypting were fixed months ago.

It only works if I use the --android tag when encrypting, I still don't know if it's the right way to do it

**Comment by Oen44** [comment]
> It only works if I use the --android tag when encrypting, I still don't know if it's the right way to do it

This is the only way to do it. If you want to encrypt files for android client, you have to add `android` argument.

**Comment by zafrim** [comment]
> > It only works if I use the --android tag when encrypting, I still don't know if it's the right way to do it
> 
> This is the only way to do it. If you want to encrypt files for android client, you have to add `android` argument.

Yes I did but my antivirus go crazy when I used this **android** argument.
What's in the libEGL.dll file? Think only Kondra knows

**Comment by Oen44** [comment]
Just google it... Its not custom, or anything fancy, its part of OpenGL framework that OTCv8 uses to render stuff.
